FAHMunge/automation.py

```diff
@@ -80,9 +82,9 @@ def merge_fah_trajectories(input_data_path, output_data_path, top_filename):
         for loading the XTC files.
 
     """
-    top = md.load(top_filename)
     n_runs, n_clones = get_num_runs_clones(input_data_path)
     for run in range(n_runs):
+        top=md.load(os.path.join(top_filename,"RUN%(run)d" % vars(),"system.pdb" ))
```

Comment:
We want to let the user specify `top_filename` that may or may not contain substitution strings like `RUN%(run)d` in it.  If you hard-code things this way, then it will break all those projects for which we specify a single reference PDB file.  Instead, we should use
```
top=md.load(top_filename % vars())
```
That way, any variables we want to substitute can be specified in the `top_filename` and they will be automatically substituted.